releases/v2.0-beta.20180319.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What&#39;s New in v2.0-beta.20180319
+toc: false
+summary: Additions and changes in CockroachDB version v2.0-beta.20180319 since version v2.0-beta.20180312
+---
+
+## March 19, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305-237-ge0b123c23.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305-237-ge0b123c23.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305-237-ge0b123c23.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305-237-ge0b123c23.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### SQL Language Changes
+
+- `NO CYCLE` and `CACHE 1` are now supported options during [sequence creation](create-sequence.html). [#23518][#23518]
+- `ISNULL` and `NOTNULL` are now accepted as alternatives to `IS NULL` and `IS NOT NULL`. [#23518][#23518]
+
+### Command-Line Changes
+
+- Changed `server.drain_max_wait` to `server.shutdown.query_wait` [#23629][#23629]
+- The generated HAProxy config has been extended with readiness checks. [#23590][#23590]
+
+### Admin UI Changes
+
+- Implemented a spinner on **Logs** page instead of saying "No Data" while loading [#23556][#23556]
+- Now using monospace font and rendering new lines on **Logs** page. Also, packed lines together more tightly. [#23556][#23556]
+- Fixed a bug in which the usable capacity of nodes was not added up correctly. [#23695][#23695]
+- Moved the **Clock Offset** graph from the **Distributed** dashboard to the [Runtime dashboard](admin-ui-runtime-dashboard.html). Now displaying each node's clock offset independently rather than aggregating them together. [#23627][#23627]
```

Comment:
Sorry, should've caught this earlier, but all links to the docs need to start with `../v2.0/` since release notes are not in a version directory. Please update all such links.